custom_components/frigate/api.py

```diff
@@ -31,9 +30,12 @@ class FrigateApiClientError(Exception):
 class FrigateApiClient:
     """Frigate API client."""
 
-    def __init__(self, host: str, session: aiohttp.ClientSession) -> None:
+    def __init__(
+        self, host: str, timezone: str, session: aiohttp.ClientSession
+    ) -> None:
         """Construct API Client."""
         self._host = host
+        self._timezone = timezone
```

Comment:
As this is only being used by a single call, perhaps we should just pass this into the recording call only and keep the client as stateless as possible?